### PR TITLE
Implement run command.

### PR DESCRIPTION
**Pull request by mareksuscak**
This PR proposes a new command that runs the service but doesn’t start it at login (nor boot).

It is based on the discussion in PR #87. 

Feedback is welcome.

### CODE REVIEW

File: cmd/brew-services.rb [hunk 1]
```diff
@@ -213,11 +221,29 @@ def cleanup
...
 
-    # Stop if loaded, then start again.
+    # Stop if loaded, then start or run again.
     def restart(target)
       Array(target).each do |service|
+        was_installed = service.started?
+        
         stop(service) if service.loaded?
-        start(service)
+        start(service) if was_installed
```
Comment (MikeMcQuaid) [inline cmd/brew-services.rb]: Make this an `if/else`

File: cmd/brew-services.rb [hunk 2]
```diff
@@ -213,11 +221,29 @@ def cleanup
...
+        run(service) unless was_installed
+      end
+    end
+    
+    # Run a service.
+    def run(target)
+      if target.is_a?(Service)
```
Comment (MikeMcQuaid) [inline cmd/brew-services.rb]: `if target.is_a?(Service) && target.loaded?`

File: cmd/brew-services.rb [hunk 3]
```diff
@@ -213,11 +221,29 @@ def cleanup
...
+        if target.loaded?
+          puts "Service `#{target.name}` already running, use `#{bin} restart #{target.name}` to restart."
+          return
+        end
+      end
+      
+      Array(target).each do |service|      
+        safe_system launchctl, "load", "-w", target.plist
+        $?.to_i.nonzero? ? odie("Failed to start `#{service.name}`") : ohai("Successfully started `#{service.name}` (label: #{service.label})")
```
Comment (MikeMcQuaid) [inline cmd/brew-services.rb]: Make this an `if/else`

**Comment by MikeMcQuaid** [comment]
Looking good so far! A few nits. May also want to add this as a test to `.travis.yml` too.

**Comment by mareksuscak** [comment]
Done. As you can tell, I'm not a Ruby guy. Anyway, removed all the nits and added a test. Maybe I should add one more to test if the `*.plist` was not installed? Also I did not squash the commits to make it easier to review. Let me know if you want me to do so afterwards.

**Comment by MikeMcQuaid** [comment]
Great work! Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**